db/migrate/20221125083555_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, foreign_key: true
```

Comment:
- [ ]  according to project requirement, kindly reference user as author i.e `t.references :author, foreign_key: { to_table: 'users' } `